src/components/Editor/EditorMenu.js

```diff
@@ -170,7 +172,8 @@ function getMenuItems(
     id: "node-menu-add-watch-expression",
     label: watchExpressionLabel,
     accesskey: watchExpressionKey,
-    click: () => addExpression(editor.codeMirror.getSelection())
+    click: () => addExpression(editor.codeMirror.getSelection()),
+    accelerator: formatKeyShortcut(L10N.getStr("expressions.key"))
```

Comment:
actually, I did more research and it seems the actual Electron API which this seems to try to emulate, actually takes unformatted strings in the form of "CmdOrCtrl+Shift+E". And does that mapping internal to the API. Which might lead to a slight duplication of the code. I guess I can refactor to expose that from a common library